src/chrome/content/rules/TouTiao.xml

```diff
@@ -0,0 +1,36 @@
+<!--
+
+	Redirect to 404error page:
+		space.toutiao.com
+	
+	Redirect to http:
+		p5a.pstatp.com
+
+	Mismatch:
+		(www.)?pstatp.com
+		i.pstatp.com
+		m.pstatp.com
+		s[4-6].pstatp.com
+		(p|v)[0-10].statp.com
+
+-->
+
+<ruleset name="toutiao.com (partial)">
+
+	<target host="toutiao.com" />
+	<target host="app.toutiao.com" />
```

Comment:
It looks like there are significant differences visiting this over `http` vs `https`